src/tests/test_error_recovery.py

```
@@ -0,0 +1,353 @@
+    assert results[0].claim == claims[1].text
+
+
+@pytest.mark.asyncio
+@patch("src.verifact_manager.Runner.run")
+async def test_invalid_verdict(mock_run, manager):
+    """Test handling of invalid verdicts."""
+    # Create test data
+    scenario = MockDataFactory.create_scenario("standard", claim_count=1)
+    claims = scenario["claims"]
+    evidence_map = scenario["evidence_map"]
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)